packages/wrangler/src/api/dev.ts

```diff
@@ -126,6 +128,12 @@ export async function unstable_dev(
 		enablePagesAssetsServiceBinding,
 	} = experimentalOptions;
 
+	const updateCheck = options?.updateCheck ?? false;
+
+	if (!options?.disableWranglerBanner) {
+		await printWranglerBanner(updateCheck);
+	}
```

Comment:
Perhaps controversial, but what about just removing this from `unstable_dev` entirely? I can't really see any reason someone would want this if they're using Wrangler as an API

Reply:
I 100% agree. The only reason I implemented the flag was because I was afraid I might be missing some context around how folks depend on this API, and didn't want to break any existing expectations. But if we're good with what you're proposing, I'm abs behind the idea!

Reply:
I had originally added this when unstable_dev was the best way to write tests, and wanted to avoid a network call when I had disabled `net.connect`. Now that we have vitest-pool-workers, I don't think it's as necessary anymore, but I'll let you decide

Reply:
awesome! thank you for providing the context ❤️

Reply:
✅